hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@pkdash @horsburgh In resource publication specification, it appears metadata only resources are not allowed to be published, and after a resource is published, no edit is possible, so I think publisher element for published resources is final and non-editable. @horsburgh correct me if I am wrong about this.